src/explorer/mod.rs

```diff
@@ -204,23 +230,23 @@ impl<'a> GitExplorer {
     }
 
     fn paint_branch(
-        &self,
+        &mut self,
         mut commits: Vec<Commit>,
         mut output: Vec<GraphNode>,
-        limit_stack: Option<usize>,
-        branches: Vec<BranchData>,
-        abort: bool) -> Vec<GraphNode> {
```

Comment:
`limit_stack` and `abort` attributes are not necessary as arguments. I can use them from the `GitExplorer` struct if I define them as attributes of that struct.